dbt_project.yml

```diff
@@ -65,6 +65,14 @@ on-run-end:
 
 models:
   snowplow_attribution:
+    +file_format: >
+      {{ 
+        var(
+          'snowplow__datalake_file_format',
+          'delta' if target.type != 'spark' else 'iceberg'
+        ) 
+      }}
+    +incremental_strategy: "{{ none if target.type not in ['spark'] else 'merge' }}"
```

Comment:
Let's discuss this offline, I thought this was still a question before I left which strategy we go for in the end out of the options on how to treat this.